tests/integration/query.ts

```diff
@@ -50,3 +58,130 @@ it('universal autocomplete sets search intents', async () => {
   expect(statefulCore.state.query.searchIntents).toEqual(['NEAR_ME']);
 });
 
+it('vertical autocomplete get latest request results', async () => {
+  const mockedCore: any = {
+    verticalAutocomplete: jest.fn(
+      async (_request: VerticalAutocompleteRequest) => {
```

Comment:
do we need to prefix request with an underscore? normally the underscore means the variable is unused